### ISSUE TITLE
Problem: Can't add level of description to directories in appraisal/arrange

### ISSUE THREAD

**Issue by jraddaoui** [archivematica/Issues#856]
**Expected behaviour**
I can enter levels of description to directories so they are respected in AtoM when the DIP is uploaded.

**Current behaviour**
The dialog appears and a level of description can be selected. However, after saving, the value is not shown on the panel (reload button doesn't fix the issue either). Randomly, when a level of description is added to all the files inside that directory, the directory displays one of the levels added to the files. Nevertheless, that (fake) level of description is not carried to the METS file, which probably means this is just an issue in the appraisal tab interface.

**Steps to reproduce**
Arrange a SIP with directories in the appraisal tab and add a level of description to a directory and then to all its files. Generate a SIP to see the final result in the METS and upload to AtoM to see that no directory is shown in there, only a flat structure of files.

**Your environment (version of Archivematica, OS version, etc)**
RC-1.10, Docker Compose environment.

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Done:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

**Comment by jraddaoui** [comment]
Tested @replaceafill's fix locally to finalize the DIP upload fixes on the AtoM side and this works as expected most of the time.

I found some edge cases when you play with nested empty directories:

* They pick the same LOD added to the descendant.
* Their directory name (and LOD) disappears.

However, if they have arranged files inside, everything works okay. Additionally, there are a couple of "bigger" issues that we may consider fixing in the future:

* Creating a dir, arranging a file or adding a LOD collapses the entire tree.
* A selected LOD can't be removed from a dir/file, only changed.
